docs/user/editions/plasma/configuration.md

```diff
@@ -1,52 +1,244 @@
+
+![Desktop context menu](configuration/desktop-context-menu.png)
+
+The following table describes how to customize the desktop using the **Desktop Folder Settings** screen.
+
+| Section                                    | Description                                                                          |
+|--------------------------------------------|--------------------------------------------------------------------------------------|
+| Change the wallpaper                       | Go to the **Wallpaper** section, choose an image from the list, and click **OK**.    |
+| Hide the desktop icons                     | Go to the **Wallpaper** section, change **Layout** to **Desktop**, and click **OK**. |
+| Configure mouse gestures                   | Go to the **Mouse Actions** section to setup mouse gestures.                         |
+| Change the folder displayed on the desktop | Go to the **Location** section, select a location in your system, and click **OK**.  |
+| Change the appearance of the desktop icons | Go to the **Icons** section, configure the necessary settings, and click **OK**.     |
+| Display only specific files in the desktop | Go to the **Filter** section,configure the necessary settings, and click **OK**.     |
+
+:::info
+**Icons** and **Location** are only available when the desktop layout is set to **Folder View**.
+:::
+
+## Virtual Desktops
+Plasma allows you to have multiple desktops on your computer. 
+
+Virtual desktops are helpful to keep tasks organized and avoid clutter. For example, you could have one virtual desktop for work, one for school, and one for personal use.
+
+Plasma arranges virtual desktops in a grid. You can configure the rows of the grid to suit your needs.
+
+:::note
+Solus Plasma has one virtual desktop and one row by default.
+:::
+
+### Creating Virtual Desktops
+1. In the [**System Settings**](#opening-system-settings) screen, go to **Workspace** > **Workspace Behavior** > **Virtual Desktops**.
+2. Define the number of rows in the grid.
+3. Create a new virtual desktop.
+   1. Click **Add**.
+   2. (Optional) Click the **Rename** button to change the name of the virtual desktop.
+4. Repeat the previous step to add multiple virtual desktops.
+
+   :::caution Important
+   Plasma arranges the desktops by the rows automatically.
+   :::
+5. (Optional) Configure the behavior of the virtual desktops.
+   - **Navigation wraps around**: Lets you cycle through your virtual desktops in a loop using keyboard shortcuts or mouse gestures.
+   - **Show animation when switching**: Displays animations when switching between virtual desktops.
+   - **Show on-screen display when switching**: Displays the name of the current desktop when switching between desktops.
+   - **Show desktop layout indicators**: Displays the virtual desktop grid when you switch between desktops.
+6. Click **Apply**
+
+:::tip
```

Comment:
It's best to add a blank line between the first and last lines of an admonition. Otherwise our linter "prettier" reformats the admonition in an invalid way. See https://docusaurus.io/docs/markdown-features/admonitions

You can see the incorrect reformatting if you view this file in preview mode:
https://github.com/getsolus/help-center-docs/blob/e1065b439556a3a52a5fad8f4ad1eb2d8d7a749b/docs/user/editions/plasma/configuration.md

So this should be 

```
:::tip

Plasma displays...

:::
```